content/blog/inngest-replit-vibe-code-ai-agents.mdx

```diff
@@ -0,0 +1,61 @@
+---
+heading: "Announcing Inngest + Replit: Vibe code your agents"
+image: /assets/blog/replit-cover.jpg
+subtitle: "Replit’s agent builder is powered by Inngest. Novice builders can now vibecode durable, complex agents quickly. Not a vibecoder? Read on to learn exactly how Replit built on top of Inngest, so you can do the same."
+date: 2025-09-25
+author: Tony Holdstock-Brown
+---
+
+*TLDR: Replit’s agent builder is now powered by Inngest! Novice builders can now vibecode durable, complex agents quickly. Not a vibecoder? Read on to learn exactly how Replit built on top of Inngest, so you can do the same.*
```

Comment:
We're inconsistent with `vibe coding` and `vibecode` - most of the google searches I see represent them as 2 works, even replit's: https://blog.replit.com/what-is-vibe-coding - matching the rest of the web is probably better for SEO